swagger/v1-prerelease/swagger.json

```diff
@@ -0,0 +1,3455 @@
+{
+  "openapi": "3.0.1",
```

Comment:
By the way, I see most docs for OpenAPI at least on their website use YAML instead. Should we by default be using YAML?

Reply:
Changed it to yaml, for the path comment, swashbuckle only supportst he paths I have currently, so 
ui: `/swagger`
yaml:` /swagger/{version}/swagger.yaml`